rust/agama-cli/src/commands.rs

```diff
@@ -72,4 +72,16 @@ pub enum Commands {
     /// not affect the root user.
     #[command(subcommand)]
     Auth(AuthCommands),
+
+    /// Download file from given URL
+    ///
+    /// Purpose of this command is to allow download from all supported schemas.
+    /// It can be used to download additional scripts, configuration files and so on.
+    /// For agama autoinstallation profiles it can be also used, but unless additional processing is required
+    /// the "agama profile import" is recommended.
+    /// For autoyast conversion use directly "agama profile autoyast".
```

Comment:
```suggestion
    /// If you want to convert an AutoYaST profile, use "agama profile autoyast".
```

Reply:
Additionally, we could put this URL to check the supported schemas: https://doc.opensuse.org/projects/autoyast/#Commandline-ay